molecule/default/tests/test_default.py

```diff
@@ -12,15 +12,40 @@
 ).get_hosts("all")
 
 
+@pytest.mark.parametrize(
+    "pkg",
+    [
+        "python-mpltoolkits.basemap",
+        "python-numpy",
+        "python-dateutil",
+        "python-netaddr",
+        "python-pandas",
+        "python-progressbar",
+        "python-docopt",
+        "python-unicodecsv",
+        "python-pypdf2",
+        "texlive",
```

Comment:
We may as well test for `rsync` as well:
```suggestion
        "rsync",
        "texlive",
```